package.json

```diff
@@ -51,8 +51,9 @@
     "tsd": "^0.31.1"
   },
   "dependencies": {
-    "fastify-plugin": "^5.0.0",
-    "cookie-signature": "^1.2.1"
+    "cookie": "^0.7.1",
```

Comment:
Can you release a 1.0.0 version?

Reply:
Seems reasonable, I can do a major version release on Monday.